File: homeworkSumPoint/Java/algB.java
import java.util.Scanner;
public class algB {
    public static void main(String[] args) {
        System.out.println("Welcome to Homework Point Sum program!");
        double sum =0;
        int itterations=0;
        while(true){
            System.out.println("Enter a homework score, or -1 to quit:");
            Scanner Score = new Scanner(System.in);
            double score = Score.nextDouble();
            if (score<0) {
                break;
            }
            sum+=score;
            itterations++;
        }
        System.out.println("Homework point sum is "+sum);
        if ((sum/itterations)<33) {
            System.out.println("Fail");
        }
    }
}

File: homeworkSumPoint/Python/algB.py
print('Welcome to the Homework Point Sum progress!')
sum=0
assignments=0
while True:
    score = int(input('Enter a homework score, or -1 to quit: '))
    if (score<0):
        break
    sum+=score
    assignments+=1
print('Homework point sum is ',sum)
if (sum/assignments<33):
    print('Fail')